exercises/practice/hello-world/hello_world.odin

```diff
@@ -0,0 +1,5 @@
+package hello_world
+
+hello_world :: proc() -> string {
+	return "Hello, World!"
```

Comment:
Should be the student stub not the example CI

Reply:
Fixed

Reply:
Looks to me like it isn't fixed?

Reply:
Hopefully fixed now that I have a test runner to tell me if it's not fixed 🙈